crates/swc_ecma_minifier/src/analyzer/mod.rs

```diff
@@ -704,6 +706,20 @@ where
         }
     }
 
+    fn visit_bin_expr(&mut self, e: &BinExpr) {
+        if e.op.may_short_circuit() {
+            e.left.visit_with(self);
+            let ctx = Ctx {
+                in_cond: true,
+                ..self.ctx
+            };
+            self.with_ctx(ctx).visit_in_cond(&e.right);
+        } else {
+            println!("else");
```

Comment:
Please remove this line

Reply:
I think this is not removed.
Maybe a lag of github?

Reply:
Oh, push failed on my side.